frontend/src/components/managePermissions/table/UserDataTable.vue

```diff
@@ -177,12 +189,14 @@ const highlightNewUserAccessRow = (rowData: any) => {
                 >
                 <Column header="Action">
                     <template #body="{ data }">
-                        <!-- Hidden until functionality is available
-                            <button
-                                class="btn btn-icon"
-                            >
-                                <Icon icon="edit" :size="IconSize.small"/>
-                            </button> -->
+                        <button
+                            title="User permission history"
+                            class="btn btn-icon"
+                            :disabled="!isDevEnvironment"
```

Comment:
Currently temporarily disabled in higher environments, only available for DEV environment.